### ISSUE TITLE
guru bug: Go/Peak to Implementation works only one way

### ISSUE THREAD

**Issue by kron4eg** [Microsoft/vscode-go#1536]
When `guru implements` used against interface, it reveals all found implementations, when used against concrete type, it reveals interfaces this type implements. But not in vscode-go.

```
package main

// Some test interface
type Some interface {
	A()
}

type a struct{}

func (*a) A() {}

type b struct{}

func (*b) A() {}

func main() {

}
```

Steps to Reproduce:

1. position cursor somewhere in line `type Some interface {`
2. call "Peek Implementation" from command line (that works)
3. position cursor somewhere in line `type a struct{}`
4. call "Peek Implementation" from command line (that doesn't work)

#8 related

**Comment by ramya-rao-a** [comment]
I have made the required changes and tested with the above code. Can you help with testing on more real world scenarios? All you need to do is
- Download https://github.com/Microsoft/vscode-go/blob/master/Go-latest.vsix
- Run `code --install-extension Go-latest.vsix` 
- If the above fails with `Error: end of central directory record signature not found`, then clone this repo and use the `Go-latest.vsix` file from the cloned repo
- Reload VS Code

**Comment by kron4eg** [comment]
@ramya-rao-a confirm, it works. Thanks a lot for such a quick fix!

**Comment by ramya-rao-a** [comment]
Thanks for confirming.
Closing this issue now.
Will be releasing an update next week

**Comment by ramya-rao-a** [comment]
This feature is now available in the latest update (0.6.78) to the Go extension
